zmon_agent/discovery/kubernetes/cluster.py

```diff
@@ -36,6 +36,7 @@ def __init__(self, region, infrastructure_account):
         # TODO: get config path from ENV variable
         self.namespace = os.environ.get('ZMON_AGENT_KUBERNETES_NAMESPACE')
         self.cluster_id = os.environ.get('ZMON_AGENT_KUBERNETES_CLUSTER_ID')
+        self.alias = os.environ.get('ZMON_AGENT_KUBERNETES_CLUSTER_ALIAS')
```

Comment:
You are right, it doesn't have to be required. Changed it so it defaults to empty string.